src/auth/create-request-storage-access.ts

```diff
@@ -38,6 +38,7 @@ export default function createRequestStorageAccess({
   <base target="_top">
   <title>Redirecting…</title>
 
+  <script src="https://unpkg.com/@shopify/app-bridge@^1"></script>
```

Comment:
We already use app bridge v1 in some other places, so I just went for consistency. Since we'll be deprecating this package soon, I felt there was no need to upgrade app bridge to the latest version since it requires some minor changes.